src/components/ResourceItem/ResourceItem.scss

```
@@ -343,3 +335,21 @@ $resource-list-item-variables: (
     background-color: var(--p-surface-selected-pressed);
   }
 }
+
+.ListItem {
+  position: relative;
+  @include focus-ring($border-width: rem(0px), $size: 'wide');
+
+  &.focused {
+    @include focus-ring($style: 'focused');
+    z-index: resource-list-item(clickable-stacking-order);
+  }
+
+  & + .ListItem {
+    border-top: border('divider');
+  }
+
+  * + ul > &:first-of-type.focused:after {
+    top: 0.1rem;
```

```suggestion
    top: rem(1px);
```

But is it safer to avoid weirdness if we remove `position: relative` and just use margin or padding?

---

I experimented with that and didn't get the same result as I expected. I can try again.